src/java/org/apache/cassandra/index/sai/disk/StorageAttachedIndexWriter.java

```diff
@@ -187,6 +194,9 @@ public void staticRow(Row staticRow, long position)
     @Override
     public void complete(SSTable sstable)
     {
+
+        tableMetrics.updateStorageAttachedIndexWritingTime(totalTimeSpent, opType);
```

Comment:
we should include the time spent in this method because `perIndexWriter.complete` involves segment flush which is time consuming.

Reply:
added, thanks